plugins/modules/template_intent.py

```diff
@@ -265,282 +525,575 @@
+                            type: str
+                          productSeries:
+                            description: Device series.
+                            type: str
+                          productType:
+                            description: Device type.
+                            type: str
+                        type: list
+                      id:
+                        description: UUID of template.
+                        type: str
+                      language:
+                        description: Template language (JINJA or VELOCITY).
+                        type: str
+                      name:
+                        description: Name of template.
+                        type: str
+                      project_name:
+                        description: Project name.
+                        type: str
+                      rollbackTemplateParams:
+                        description: Configuration Template Import Template's rollbackTemplateParams.
+                        elements: dict
+                        suboptions:
+                          binding:
+                            description: Bind to source.
+                            type: str
+                          customOrder:
+                            description: CustomOrder of template param.
+                            type: int
+                          dataType:
+                            description: Datatype of template param.
+                            type: str
+                          defaultValue:
+                            description: Default value of template param.
+                            type: str
+                          description:
+                            description: Description of template param.
+                            type: str
+                          displayName:
+                            description: Display name of param.
+                            type: str
+                          group:
+                            description: Group.
+                            type: str
+                          id:
+                            description: UUID of template param.
+                            type: str
+                          instructionText:
+                            description: Instruction text for param.
+                            type: str
+                          key:
+                            description: Key.
+                            type: str
+                          notParam:
+                            description: Is it not a variable.
+                            type: bool
+                          order:
+                            description: Order of template param.
+                            type: int
+                          paramArray:
+                            description: Is it an array.
+                            type: bool
+                          parameterName:
```

Comment:
are we still using camelcase?

Reply:
Export and import template and projects are blocked so we are using camel case similar to one to one API.